rebasehelper/application.py

```diff
@@ -439,6 +440,9 @@ def build_packages(self):
             results_dir = os.path.join(self.results_dir, version)
             build_dict['builds_nowait'] = self.conf.builds_nowait
             build_dict['build_tasks'] = self.conf.build_tasks
+            build_dict['enable_option'] = self.conf.enable_option
+            if self.conf.buildtool in ("copr", "fedpkg"):
+                logger.warning("We are not supporting option --enable-option for builder copr or fedpkg.")
```

Comment:
When user uses copr/fedpkg it does not make sense to inform them about some option they didn't even specified on command line.

The only situation when it would make sense it when they explicitly use copr/fedpkg with the option.